### PR TITLE
LSM: Update comment in Groove to reflect code changes

### PR DESCRIPTION
**Pull request by ifreund**

### CODE REVIEW

**Review by sentientwaffle — DISMISSED** [review]
Thanks for the clarification!

**Comment by ifreund** [comment]
@sentientwaffle I've just fixed a small typo and rebased onto main, but it looks like GitHub requires a fresh press of the Approval button before this can be merged if you don't mind :)

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**
